lambdatimes/src/components/Content/Content.js

```diff
@@ -1,56 +1,54 @@
 import React, { Component } from 'react';
-
 import Tabs from './Tabs';
 import Cards from './Cards';
-
-// Importing our tab and card data. No need to change anything here.
 import { tabData, cardData } from '../../data';
+import Styled from 'styled-components';
+
+const ContentContainer = Styled.div`
+  display: flex; 
+  flex-direction: column; 
+  align-items: center;
+`
 
 export default class Content extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      selected: 'all',
+      selectedTab: 'all',
       tabs: [],
       cards: []
     };
   }
 
   componentDidMount() {
-    // Once the component has mounted, get the data and reflect that data on the state.
+    this.setState({tabs:tabData,cards:cardData})
```

Comment:
consider putting spaces inside objects, colons, and after commas, as this increases legibility
this.setState({ tabs: tabData, cards: cardData })